src/plugins/dashboard/common/content_management/v1/types.ts

```diff
@@ -49,8 +49,14 @@ export interface SavedDashboardPanel {
   panelRefName?: string;
   gridData: GridData;
   panelIndex: string;
-  version: string;
   title?: string;
+
+  /**
+   * This version key was used to store Kibana version information from versions 7.3.0 -> 8.11.0.
+   * As of version 8.11.0, the versioning information is now per-embeddable-type and is stored on the
+   * embeddable's input. (embeddableConfig in this type).
```

Comment:
Yes exactly. The version key will no longer exist in the Dashboard panels themselves. It will exist in the `input` and only if the Embeddable factory has a `latestVersion` defined.